packages/app/src/app/store/modules/profile/actions.js

```diff
@@ -13,7 +13,7 @@ export function deleteSandbox({ api, state }) {
         id: sandboxId,
       },
     })
-    .then(() => undefined);
+    .then(() => state.set(`profile.current.sandboxCount - 1`));
```

Comment:
I think that this one is still from the other PR.

Reply:
I think github recognises it won't be a problem, If you want I can create new commit